.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,46 @@
+name: Node.js Continuous Integration Smoke Test
+
+on:
+  pull_request:
+    branches: [master]
```

Comment:
I was about to wonder about unit tests but see the file called `pr-build.yml` runs on main branch push :P